atecc608a/main.c

```diff
@@ -76,18 +76,32 @@
     "the slot now behaves according to the policies set by the associated\n"\
     "configuration zone’s values. [y/n]: "
 
+static void psa_purge_keys(void)
+{
+    psa_key_id_t id;
+    printf("Clearing persistent key data\n");
+    /* Clear slots 0-15 of persistent storage. These numbers are equal to
+     * physical slots available on the device. */
+    for (id = 0; id < 16; id++) {
+        psa_destroy_persistent_key(id);
+    }
+    /* Purge the transaction file. */
+    psa_crypto_stop_transaction();
+}
```

Comment:
This function is yucky to put into an example, which should be more whitebox style (what users should actually be recommended to do and can use).

Reply:
We have to compromise between showing an example and testing somehow. Purging the storage is needed.

Reply:
`mbed_psa_reboot_and_request_new_security_state(PSA_LIFECYCLE_ASSEMBLY_AND_TEST); ` will purge storage in a slightly more acceptable way